src/templates/supported.html

```diff
@@ -6,22 +6,53 @@
+        <li>CubingUSA can cover event insurance if required by the venue.</li>
+      </ul>
+      <h3>Requirements Highlights</h3>
       <p>
-        Organizers must submit an application including a budget and other competition details at least <b>60 days in advance</b>.  Income and expenses need to be pre-approved, and generally should be paid or received by CubingUSA.  In order ensure that non-profit laws are followed, there are limitations to what expenses a competition may pay.
+        In order to ensure financial accountably, organizers must follow policies which include:
       </p>
+      <ul>
+        <li>Organizers must submit an application including an estimated budget at least <b>60 days in advance</b>.</li>
+        <li>Any net income goes into the CubingUSA general fund.</li>
+        <li>Organizers must agree to the CubingUSA Code of Conduct as seen on the <a href="{{ c.uri_for('documents') }}">Documents</a> page.</li>
+        <li>Notably, organizers may not personally financially benefit and must report any potential conflicts of interest.</li>
+        <li>Expenses must be pre-approved and may only be re-imbursed with receipts.</li>
+        <li>Expenses that were not pre-approved may be rejected at CubingUSA's discretion.</li>
+        <li>All sponsorships, partnerships, and prizes must be pre-approved.</li>
+        <li>A CubingUSA Deputy must be involved in the organization of the competition and must be present at the competition to ensure relevant policies are followed. (CubingUSA Deputies may or may not also be an organizer or WCA Delegate.)</li>
+        <li>A Financial Report must be submitted after the competition.</li>
+      </ul>
+      <h3>Full Policy</h3>
       <p>
-        A <b>Deputy</b> appointed by the CubingUSA Board of Directors must participate in the organization of the competition, and must be present at the competition.  The Deputy is responsible for ensuring all relevant policies are followed, and for reporting any potential issues to the Board of Directors.
+        The full Supported Competitions Policy is available on our <a href="{{ c.uri_for('documents') }}">Documents</a> page.
       </p>
+      <h3>More Information</h3>
       <p>
-        The full policy is available on our <a href="{{ c.uri_for('documents') }}">Documents</a> page.
+        If you are interested in organizing a CubingUSA Supported Competition, please use our <a href="{{ c.uri_for('contact') }}">contact page</a> for more information.
       </p>
-      <h3>More Information</h3>
+      <h3>How to file an expense reimbursement request</h3>
       <p>
-        If you are interested in organizing a CubingUSA Supported Competition, please contact your WCA Delegate or the <a href="{{ c.uri_for('contact') }}">CubingUSA Board</a> for more information.
+        Send an email to treasurer(at)cubingusa.org with the following information:
```

Comment:
Do you think including the email address is necessary here?  The organizer will already have a thread with the directors, who can tell them what to do with receipts.  Could we just say "the CubingUSA treasurer"?

Mostly I ask because there aren't any email addresses currently on our website, and I'm a little skeptical that spammers aren't going to notice "foo(at)bar.com"-type email addresses.

Reply:
Sounds good.